src/test/java/com/klarna/hiverunner/HiveServerContainerTest.java

```diff
@@ -94,12 +94,8 @@ public void testTearDownShouldNotThrowException() {
         container.tearDown();
     }
 
-    @Test(expected = HiveSQLException.class)
+    @Test
     public void testInvalidQuery() throws Throwable {
-        try {
-            container.executeStatement("use foo");
-        } catch (IllegalArgumentException e) {
-            throw e.getCause();
-        }
+        Assertions.assertThrows(IllegalArgumentException.class, () -> container.executeStatement("use foo"));
```

Comment:
Mmmm, this isn't testing quite the same thing. We need to call `getCause` on the `IllegalArgumentException` and assert that it's a `HiveSQLException`.